### DISCUSSION TITLE
MAA打开后过了一会儿就自动关闭了，没有任何报错

### DISCUSSION THREAD

**Discussion by xueqi5201314** [MaaAssistantArknights/MaaAssistantArknights#7902] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/108047693/1944c3c5-eb74-4136-833f-7ec77eb36f36)
我在事件查看器里面看到错误
系统信息
版本	Windows 11 专业版
版本	23H2
安装日期	‎2023/‎6/‎12 ‎星期一
操作系统版本	22635.2841
体验	Windows Feature Experience Pack 1000.22681.1000.0
安装好了net 8
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/108047693/0857654f-5b27-4cdb-bc74-c9bf701c100c)

**Comment by ABA2396** [comment, marked as answer]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
